src/classifier/Alphabet.h

```diff
@@ -83,9 +83,9 @@ class Alphabet : public std::tr1::unordered_map <string, int> {
     }
   }
 
- private:
+private:
```

Comment:
Microsoft Visual Studio does not allow that in the formatting document options. I can either indent access specifiers to the class or struct identifier (therefore giving it the defined indentation size: 4 spaces), or not.